### PR TITLE
Added assignable event items

### PR DESCRIPTION
**Pull request by delroyhiebert**

### CODE REVIEW

**Comment by JenCabral** [comment]
Looks straight forward. :+1:

File: partEZ/app/Models/EventListItem.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 use DB;
 use Illuminate\Database\Eloquent\Model;
+use DB;
```
Comment (minnerthecat) [inline partEZ/app/Models/EventListItem.php:6]: DB is already included above

**Comment by minnerthecat** [comment]
Aside from the extra include, it looks good!

**Comment by delroyhiebert** [comment]
Removed the include!

**Pull request merged**